script/travis/print_valgrind_log.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+if [[ "${DYNAMIC_ANALYSIS}" == "ON" ]]; then
```

Comment:
Shouldn't it be like `if [[ "$DYNAMIC_ANALYSIS" == "ON" ]]; then` without `{}`?

Reply:
25 commits on this PR :). Believe me, I tried every single combination. Also every combination of `after_success` and alike. Let's wait 5 minutes and see if my last change helped.

Reply:
You could set the number of iterations to 10 (it would be even faster). I can cancel previous compilations if you push multiple commits.

Reply:
Still no success at printing valgrind log.